config/500-controller.yaml

```diff
@@ -34,10 +34,11 @@ spec:
       containers:
       - name: controller
         image: github.com/GoogleCloudPlatform/cloud-run-events/cmd/controller
+        imagePullPolicy: Always
```

Comment:
need Always here?

Reply:
yeah or else it does not pull latest

Reply:
I think this goes along with the ko comment I made earlier. Once we have releases for these, then I'd expect we could change this from Always.